src/smt/smt_engine.cpp

```diff
@@ -1489,17 +1467,8 @@ Result SmtEngine::checkSynth()
 
     d_private->d_sygusConjectureStale = false;
 
-    if (options::incrementalSolving())
-    {
-      // we push a context so that this conjecture is removed if we modify it
-      // later
-      internalPush();
-      assertFormula(body, true);
-    }
-    else
-    {
-      query = body.toExpr();
-    }
+    // TODO (project #7): if incremental, we should push a context and assert
+    query = body.toExpr();
```

Comment:
It was implemented at a placeholder but not tested (and now abandoned since we hit other snags when trying to do SyGuS+incremental).  The implementation was not ideal since it was making a low level call to `internalPush` which is now a private method of SmtEngineState.  If we implement this in the future, we should figure out how to do this cleanly.